### ISSUE TITLE
Should I translate README.md

### ISSUE THREAD

**Issue by iamthesenate1** [FontManager/font-manager#190]
I know this is not exactly an issue but I wondered if you allow different language README files?
I saw other projects do this but I don't know if you do this too.
If yes, I'd be willing to translate in romanian.

**Comment by JerryCasiano** [comment]
I have no issue with this. 

Sounds like a good thing to me.

I think where to place it and how to name it are worth sorting out though, in case others would like to do the same thing.

And do we link to them in main README.

**Comment by iamthesenate1** [comment]
I saw NewPipe do this with README.language.md files and than linking them in the main README: https://github.com/TeamNewPipe/NewPipe
Here is another thread about the same thing: https://github.com/adalbertobrant/metabolismobasal/issues/7.
From what I understand Github doesn't have an official way of doing it.

Anyway, I think I'll finish translating by tomorrow.

**Comment by iamthesenate1** [comment]
I'm thinking that it should look something like this:

Read this in other languages: [`Română`](https://github.com/FontManager/font-manager/blob/master/README.ro.md)

What's your opinion?

**Comment by JerryCasiano** [comment]
> What's your opinion?

I think the way NewPipe handled this is probably the way to go. So yes, "Read this in other languages:" sandwiched between <hr> and starting with the English version followed by any available translations in their native name I think is the way to go.

Can you add that to your PR?

**Comment by iamthesenate1** [comment]
> Can you add that to your PR?
> 
> If not, I can update the README but I need the native name (spelling) for the language.

The native spelling is Română.
I tried to do it myself, but I couldn't get the anchor links to work between <hr>.

**Comment by iamthesenate1** [comment]
> Can you add that to your PR?
> 
> If not, I can update the README but I need the native name (spelling) for the language.
The native spelling is Română.
I tried to do this myself but I couldn't get the anchor links to work between `<hr>`.

**Comment by JerryCasiano** [comment]
> The native spelling is Română.

Yeah, I realized after I posted that you had already provided what I needed in your previous comment...

> I tried to do this myself but I couldn't get the anchor links to work between <hr>.

No problem. I just updated it. Have a look when you get a moment. I just used a table at the top. I think it looks ok.

I skipped the "Read this in other languages" part because it seemed redundant. 

I'm open to changes if you have any other suggestions on this.

Thank you for taking the time to help out with this.  :-)
